firebase/db_operate.py

```diff
@@ -0,0 +1,40 @@
+import datetime
+import os
+
+import firebase_admin
+from dotenv import load_dotenv
+from firebase_admin import credentials, db, firestore
+
+load_dotenv()
+
+cred = credentials.Certificate("firebase/cred.json")
+
+firebase_admin.initialize_app(
+    cred,
+    {
+        "databaseURL": os.getenv("FIREBASE_URL"),
+        "databaseAuthVariableOverride": {"uid": "my-service-worker"},
+    },
+)
+
+fireDB = firestore.client()
+
+
+def create_item(sente, gote) -> None:
```

Comment:
- 型アノテーション付けて欲しいです
- ID返して欲しいです（サーバーのレスポンスとしてID返してほしいので）
```suggestion
def create_item(sente: str, gote: str) -> str:
```